contracts/test/DeployLightClientTestScript.s.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+
+import { LightClientTest as LCTest } from "./mocks/LightClientTest.sol";
+import { LightClient as LC } from "../src/LightClient.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployLightClientTestScript is Script {
+    function run(uint32 numBlocksPerEpoch, uint64 numInitValidators, address owner)
+        external
+        returns (address payable proxyAddress, address admin, LC.LightClientState memory)
+    {
+        // TODO for a production deployment provide the right genesis state and value
```

Comment:
Do we still need this todo? If yes, would it be better to have a github issue since this file isn't something that would be used in production.

Reply:
Yes we do, issue created here https://github.com/EspressoSystems/espresso-sequencer/issues/1346